src/Assignment.ts

```diff
@@ -0,0 +1,59 @@
+import Question from "./model/Question";
+import alasql from "alasql";
+
+const question = new Question(
+    // Prompt for the user
+    `Select all columns from the "Employees" table.`,
+
+    // SQL queries to configure the context
+    `CREATE TABLE Employees (ID INT, Name VARCHAR(255), Supervisor INT);
+    INSERT INTO Employees VALUES
+        (1, "Joe", NULL),
+        (2, "Bill", 1),
+        (3, "Jill", NULL),
+        (4, "Bob", 3),
+        (5, "Marvin", 1);`,
+
+    // Answer key query
+    `SELECT * FROM Employees`);
+
+function compareResults(a: Array<any>, b: Array<any>) {
+    if (a.length != b.length) return false;
```

Comment:
Use strict equality !==

Type coercion still happens in TS since the typing is pretty lose so we can't drop strict equality checks. Probably wouldn't be an issue in this instance but better to always use it if you're expecting the types to match.